gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/Message.java

```diff
@@ -244,17 +244,24 @@ public ByteArray payload() {
     return payload;
   }
 
+  final boolean baseEquals(Message message) {
+    return Objects.equals(id, message.id)
+        && Objects.equals(payload, message.payload)
+        && Objects.equals(attributes, message.attributes)
+        && Objects.equals(publishTime, message.publishTime);
+  }
+
   @Override
   public boolean equals(Object obj) {
     return obj == this
         || obj != null
         && obj.getClass().equals(Message.class)
-        && Objects.equals(toPb(), ((Message) obj).toPb());
+        && baseEquals((Message) obj);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(serialVersionUID, id, payload, attributes, publishTime);
+    return Objects.hash(id, payload, attributes, publishTime);
```

Comment:
No, I did not and +1 for removing it. Was probably done when I was experimenting with one of the IntelliJ hashCode generator based on Java 7 Objects file.